src/components/cart/cart.js

```diff
@@ -0,0 +1,17 @@
+import { useData } from "../../context/data-context";
+import {CartItem} from "./cartItem";
+import { EmptyCart } from "./emptyCart";
+
+export function Cart() {
+  const { data } = useData();
```

Comment:
```suggestion
  const { data: { cartList } } = useData();
```
 we can destructure data object here for our ease.